packages/ember-engines/package.json

```diff
@@ -102,20 +108,16 @@
     "broccoli-merge-trees": "^4.2.0",
     "calculate-cache-key-for-tree": "^2.0.0",
     "ember-asset-loader": "^1.0.0",
-    "ember-cli-babel": "^7.18.0",
+    "ember-cli-babel": "^7.26.10",
+    "ember-cli-htmlbars": "^6.3.0",
     "ember-cli-preprocess-registry": "^3.3.0",
     "ember-cli-string-utils": "^1.1.0",
     "ember-cli-version-checker": "^5.1.2",
     "lodash": "^4.17.11"
   },
   "peerDependencies": {
-    "ember-source": "^3.24.1 || 4 || 5",
-    "@ember/legacy-built-in-components": "*"
-  },
-  "peerDependenciesMeta": {
-    "@ember/legacy-built-in-components": {
-      "optional": true
-    }
+    "ember-engines-router-service": ">=0.5.0",
```

Comment:
@void-mAlex if I'm reading `packages/ember-engines/addon/components/link-to-external.js` correctly, it does not depend on `ember-engines-router-service`.

am I get it right that only reason for this peer dependency - we delete `transitionToExternal`/`replaceWithExternal` route methods and `transitionToExternalRoute` controller methods because they remove in v5 as explained in https://deprecations.emberjs.com/v3.x#toc_routing-transition-methods?

If so, I think this needs to be extracted to separate PR as it's own deprecation. I can do that

Reply:
also, why don't we make `ember-engines-router-service` a dependency of `ember-engines` so ppl don't need to install it themselves?

Reply:
> also, why don't we make `ember-engines-router-service` a dependency of `ember-engines` so ppl don't need to install it themselves?

simply because it logically doesn't have enough enough knowledge to handle external routes if it was scoped to only the engine
it's an app concern around it's routes and we shouldn't force it into peoples' builds that don't use routable engines

ps I'll try and answer the first comment a bit later

Reply:
> we shouldn't force it into peoples' builds that don't use routable engines

sounds like this should be optional peer dependency then?

Reply:
> @void-mAlex if I'm reading `packages/ember-engines/addon/components/link-to-external.js` correctly, it does not depend on `ember-engines-router-service`.

correct

> am I get it right that only reason for this peer dependency - we delete `transitionToExternal`/`replaceWithExternal` route methods and `transitionToExternalRoute` controller methods because they remove in v5 as explained in https://deprecations.emberjs.com/v3.x#toc_routing-transition-methods?

also correct and it should be optional as you previously stated

> If so, I think this needs to be extracted to separate PR as it's own deprecation. I can do that

ty

Reply:
@void-mAlex so `@ember/legacy-built-in-components` is still listed in `peerDependenciesMeta`, I think you want to replace it with `ember-engines-router-service` as we want `ember-engines-router-service` be optional and it's not a requirement

Reply:
nice catch, ty